helm/cluster-api-provider-aws/values.yaml

```diff
@@ -4,23 +4,14 @@ name: cluster-api-provider-aws
-#   * default instance warmup field addition (https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/4760)
-#   * do not place EKS nodes in the CNI subnets (https://github.com/giantswarm/cluster-api-provider-aws/pull/587)
-#   * add ID to secondary subnets in AWSCluster (https://github.com/giantswarm/cluster-api-provider-aws/pull/589)
-#   * add non root volumes to AWSMachinePools (https://github.com/giantswarm/cluster-api-provider-aws/pull/591)
-#   * support adding custom secondary VPC CIDR blocks in `AWSCluster` (https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/4898)
-#   * S3 storage for user data
-#     * Support S3 bucket storage for user data of `AWSMachinePool` (https://github.com/giantswarm/cluster-api-provider-aws/pull/592)
-#     * Delete machine pool user data files that did not get deleted yet by the lifecycle policy (https://github.com/giantswarm/cluster-api-provider-aws/pull/593)
-#     * Test fixes (https://github.com/giantswarm/cluster-api-provider-aws/pull/596)
-#     * Cancel instance refresh on any relevant change to ASG instead of blocking until previous one is finished (which may have led to failing nodes due to outdated join token) (https://github.com/giantswarm/cluster-api-provider-aws/pull/598)
-#     * Use feature gate for S3 storage (https://github.com/giantswarm/cluster-api-provider-aws/pull/599)
-#     * Continue reconciliation after filling an empty `AWSMachinePool.status.launchTemplateVersion` field (https://github.com/giantswarm/cluster-api-provider-aws/pull/602)
-#     * Try deleting machine pool user data file from S3 when pruning an old launch template version (https://github.com/giantswarm/cluster-api-provider-aws/pull/601)
-tag: v2.4.2-gs-70b4664c1
+# Changes in this tag include upstream v2.5.2 with additional:
+#   * GiantSwarm fork modifications (https://github.com/giantswarm/cluster-api-provider-aws/pull/607/commits/08280ec433b63119d652b39c06d9a4b78d5a9055)
+#   * when aws-cni is disabled deleted all resources including helm managed resources (https://github.com/giantswarm/cluster-api-provider-aws/pull/607/commits/bbbb36a7a7bd050bb232bfa60a10b4202ac99850)
+#   * do not place EKS nodes in the CNI subnets (https://github.com/giantswarm/cluster-api-provider-aws/pull/607/commits/edd14fb6bd39434a249ed8696e0abeb9b85f6c35)
+#   * add non root volumes to AWSMachinePools (https://github.com/giantswarm/cluster-api-provider-aws/pull/607/commits/ab0528f4b80ed36fd7b508c4137ee09d41fefa8c)
+#   * support adding custom secondary VPC CIDR blocks in `AWSCluster` (https://github.com/giantswarm/cluster-api-provider-aws/pull/607/commits/ae00674b1ed47e0e939f44d4712507839df97106)
+#   * S3 user data support for AWSMachinePools (https://github.com/giantswarm/cluster-api-provider-aws/pull/607/commits/905c44ab2afde5e2d4e983066ab9856859a0de47)
+tag: v2.5.2-gs-905c44ab2
```

Comment:
We should first merge https://github.com/giantswarm/cluster-api-provider-aws/pull/607 and then tag the merge commit. I'm looking into it.

LGTM until here.

Reply:
> We should first merge [giantswarm/cluster-api-provider-aws#607](https://github.com/giantswarm/cluster-api-provider-aws/pull/607) and then tag the merge commit. I'm looking into it.

Definitely.

Reply:
Done